[tls] Fix: Old LBs can have TLS listeners/pools without tls_versions/tls_ciphers

BenjaminLudwigSAP opened the pull request.

BenjaminLudwigSAP committed:
[tls] Fix: Existing LBs can have TLS listeners/pools without tls_versions

notandy commented:
So I've tested this branch and found following issues:

### ssl3Enable unsupported

ssl3Enabled is only supported from AS3 3.33 upwards, see here: https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/tls-encryption.html#disablessl

```
Support for disabling SSL on TLS profiles is available in BIG-IP AS3 v3.33 and later.
```
So this fails with `    "message": "Invalid data property: ssl3Enabled"`, 

So either implement a version check or comment it out for now.

### pool and listener tls_ciphers can be also null, and need to be set to default in that case.

BenjaminLudwigSAP committed:
[tls] Remove SSL3: Unneeded and not supported in all AS3 versions

SSL3 was added for completeness in #178.

Since it's not supported in all AS3 versions[1] and we don't need it,
it's better to remove it again instead of adding unnecessary version
checking code.

[1] https://github.com/sapcc/octavia-f5-provider-driver/pull/181#issuecomment-1274962592

BenjaminLudwigSAP committed:
[tls] Fix: Old LBs can have TLS listeners/pools without tls_ciphers

Closes #179
